### ISSUE TITLE
Colour-blindness friendly VU-meter

### ISSUE THREAD

**Issue by reneknuvers** [jamulussoftware/jamulus#2260]
Current VU (signal level) indicators use green, yellow and red. 9 distinct levels are shown in the 'FANCY' theme, and a continuous bar in the other themes. No indication is given about the real level (in dBFS for example) next to the meter. One or two yellow is the assumed 'target zone' for a decent input level without distortion.

People with certain types of colour blindness have problems distinguising yellow and green, and maybe even red. To overcome this problem, and possibly for better understanding at what to aim for with the input gain knob, an indication next to the meters could be added that specifies the 'target area'.

VU-meters are used in several themes. Per theme the solution could be adopted to accommodate the theme. The use of specific colours for the marking would defeat the purpose. On old style needle-meters in tape decks the saturation area or target area was often marked by a red, black or white (contrasting to the background) area between the tick marks ( [ref 1](https://www.amazon.nl/POFET-versterker-mengkraan-mengbatterij-achtergrondverlichting/dp/B086YGG2FP/ref=asc_df_B086YGG2FP/?tag=nlshogostdde-21&linkCode=df0&hvadid=454776978703&hvpos=&hvnetw=g&hvrand=18254182795733842128&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9064872&hvtargid=pla-920777866301&psc=1), [ref 2](https://www.google.nl/imgres?imgurl=https%3A%2F%2Fmedia.istockphoto.com%2Fvectors%2Fanalog-electronic-vu-meter-vector-id480931181&imgrefurl=https%3A%2F%2Fwww.istockphoto.com%2Fnl%2Fvector%2Fanalog-electronic-vu-meter-gm480931181-36688924&tbnid=Qz4tXtW2RMn-HM&vet=12ahUKEwin5ZOd-sL1AhWI66QKHYkjALsQMygUegUIARDDAw..i&docid=TpR56Id4L12eWM&w=1024&h=696&itg=1&q=VU%20meter&hl=nl&ved=2ahUKEwin5ZOd-sL1AhWI66QKHYkjALsQMygUegUIARDDAw)

VU-meters are used in both the own input meter and next to external feeds in the mixing desk. An easy to comprehend view that is coherent in both situaties would be best.

**Comment by pljones** [comment]
> No indication is given about the real level (in dBFS for example) next to the meter.

There's a good reason for that.  It's extremely approximate so no numeric output would be particularly meaningful.  The height of LEDs is sufficiently indicative.   Nearer the top means louder.  If the signal clips, you get visual feedback.

Well over half way up and sufficiently below clipping (what would show as hitting yellow quite a lot if you've got a scheme with the colours) is "good".

**Comment by reneknuvers** [comment]
@pljones The issue is not about adding a dB-scale to it. The issue is that people with limited colourvision have no visual clue about what level they should hit or not. I ran into this problem with a respectable sound engineer that uses my server, but was clueless about the colours simply because of his disability to see colours.

**Comment by pljones** [comment]
The colour isn't relevant.  Only the height.

**Comment by reneknuvers** [comment]
I disagree. The colour is the reference for the acceptable level. What else is hinting the acceptable level? Impaired people should just ‘know’ that six up is okay and eight up is kinda bad?

**Comment by pljones** [comment]
Not all the skins have different colours, so it's an option that already exists.  The colours, when in use, are in no way definitive, either.  Top third is only a guide.  Loud enough without clipping.  And it depends how loud everyone else is, too.

**Comment by gilgongo** [comment]
We did do some work on the LED colour contrast on the fancy skin for accessibility reasons. 

>  I ran into this problem with a respectable sound engineer that uses my server, but was clueless about the colours simply because of his disability to see colours. 

Somewhat OT, but I think this is also an example of a problem Jamulus has in presenting itself as a skeuomorph of a mixing desk.

**Comment by DavidSavinkoff** [comment]
I'm not colour blind but my laptop is. Maybe the colours could be user configurable.
My problem is with the two shades of green/mud for ping time in Connect setup.

**Comment by gilgongo** [comment]
Do they have sufficient visual contrast to be distinguishable? If so, then your problem is an aesthetic one, no?

(BTW moving this to a discussion until we can agree on some actionable specification)
